### ISSUE TITLE
Jumping to Overlays in "accessibility context" missies the real value of AI in Design and Developement

### ISSUE THREAD

**Issue by UsableNet-WCAG** [w3c/ai-accessibility#21]
The current section jumps too quickly into highlighting overlays as a key way AI delivers value to web content. This framing overlooks a vastly more impactful area where AI is already being used—and should be further developed: empowering design and development workflows, especially in CMS and templated site publishing.

There are millions of designers and developers creating websites, along with major CMS platforms pushing out entire site experiences daily. (250,000 a day globally) In contrast, only two or three overlay vendors are claiming AI will revolutionize accessibility through their narrowly scoped tools. That’s a disproportionate emphasis.

We should encourage a shift in focus toward how AI can empower the hundreds of thousands of professionals building the web and apps today. From AI-powered copilot tools in development environments to intelligent plug-ins within popular CMS platforms, this is where AI can truly scale accessibility impact.

This is the information that matters most to the product managers of Figma, Shopify etc,  and CMS platform teams. Empowering these professionals with AI-driven accessibility knowledge and the possibilities it could bring has the potential to reshape accessibility across the web in a way overlays never will.

Framing overlays as the prime example of AI’s future in accessibility is akin to the tail wagging the dog.

**Comment by aardrian** [comment]
Agreed, and not just because overlays themselves have generally failed to integrate "AI" or, when they have tried, have done it poorly (I touch on some of this in #16 ).

**Comment by UsableNet-WCAG** [comment]
I have noticed an updated editors note that seems to have taken some recent issues into account is much appreciated. Aspiring to engage AI before the fact (deployment of websites) as the editors notes indicate - rather than focused on after the fact (evaluation and overlays) - will have far more impact .

**Comment by joshueoconnor** [comment]
Discussed on RQTF - seems to be addressed in the new structure.

**Comment by aardrian** [comment]
I disagree this is addressed. For example:

> the applicability of an overlay that contributes accessibility improvements is similar to the use of AI chatbots and other prompting mechanisms, […]

This is a false equivalency. An LLM chatbot user generally has a specific outcome in mind -- playing with writing styles, generating a piece of code to solve an issue, making pictures, simplifying language, etc. One gives the user agency, the other does not.

> […] thereby suggesting this may prove to be another area where generative AI could introduce improvements.

That conclusion is based on the prior false equivalency. If the end user has no idea what to fix (perhaps they cannot perceive a control or recognize how a widget is broken), then they have to rely on a third-party vendor having a pre-defined "prompt" (to carry the metaphor forward) and hope it works. For each visit / page load.

Generative AI, where it works, should fix issues as infrequently (and consistently) as possible and without putting the burden on the user (and device). Doing this at code creation or on the server would do that. For customization that the user wants on top of that, that would be best provided by tools on their OS or in their browser so it can apply to all sites and potentially learn. In other words, better outcomes can be achieved with generative AI by moving it out of overlays and to the sites themselves and/or to the OS/browser.

Overlays are almost literally the worst place to deploy the features being described.
